.github/workflows/ci-unwelcome-words.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.sha }}
+    - name: Checkout tool
+      uses: actions/checkout@v3
+      with:
+        repository: exactpro-th2/ci-github-action
+        ref: master
+        token: ${{ secrets.PAT_CI_ACTION }}
+        path: ci-github-action
```

Comment:
What is this repository for? What does this CI action? Could you please make the action name more clear to indicate what this action does?